moveos/moveos-stdlib/moveos-stdlib/sources/object.move

```diff
@@ -141,20 +141,73 @@ module moveos_std::object {
     /// Transfer ownership of `obj` to `recipient`. `obj` must have the
     /// `key` attribute, which (in turn) ensures that `obj` has a globally
     /// unique ID.
-    public fun transfer<T: key>(obj: T, recipient: address) {
+    public fun transfer<T: store>(obj: Object<T>, recipient: address) {
         // TODO: emit event
         transfer_internal(obj, recipient)
     }
 
     /// Freeze `obj`. After freezing `obj` becomes immutable and can no
     /// longer be transferred or mutated.
-    public native fun freeze_object<T: key>(obj: T);
+    //public fun freeze_object<T: store>(obj: Object<T>);
 
     /// Turn the given object into a mutable shared object that everyone
     /// can access and mutate. This is irreversible, i.e. once an object
     /// is shared, it will stay shared forever.
-    public native fun share_object<T: key>(obj: T);
+    //public native fun share_object<T: store>(obj: Object<T>);
+
+    native fun transfer_internal<T: store>(obj: Object<T>, recipient: address);
+
+    // Box style object
+     struct Object<T: store> {
```

Comment:
In both core Move and Sui Move, the [key ability] denotes a type that can appear as a key in global storage. Does box style Object need the [key ability]  , and the [store ability]?

Reply:
No. `Object` has no `key` or `store`, so the developer can not put Object to global storage(move_to), or embed to another struct. only the `transfer` function can consume the Object.